Sources/generate-swift-syntax-builder/Templates/BuildableNodesFile.swift

```diff
@@ -0,0 +1,372 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+let buildableNodesFile = SourceFile {
+  ImportDecl(
+    leadingTrivia: .docLineComment(copyrightHeader),
+    path: "SwiftSyntax"
+  )
+
+  for node in SYNTAX_NODES where node.isBuildable {
+    let type = node.type
+    let baseType = node.baseType
+    let hasTrailingComma = node.traits.contains("WithTrailingComma")
+    let conformances = [baseType.buildableBaseName, type.expressibleAsBaseName] + (hasTrailingComma ? ["HasTrailingComma"] : [])
+
+    // Generate node struct
+    StructDecl(
+      leadingTrivia: node.documentation.isEmpty
+        ? []
+        : .docLineComment("/// \(node.documentation)") + .newline,
+      modifiers: [TokenSyntax.public],
+      identifier: type.buildableBaseName,
+      inheritanceClause: createTypeInheritanceClause(conformances: conformances)
+    ) {
+      let children = node.children
+
+      // Generate members
+      for child in children {
+        VariableDecl(.let, name: child.swiftName, type: child.type.buildable)
+      }
+
+      VariableDecl(
+        leadingTrivia: [
+          "/// The leading trivia attached to this syntax node once built.",
+          "/// This is typically used to add comments (e.g. for documentation).",
+        ].map { .docLineComment($0) + .newline }.reduce([], +),
+        .let,
+        name: "leadingTrivia",
+        type: "Trivia"
+      )
```

Comment:
Nitpick: Since the leading trivia is displayed before the first node, what do you think about moving it in front of the children variables?